index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en" class=" csopmgezmow idc0_326">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <script src="https://kit.fontawesome.com/aa7a31f950.js" crossorigin="anonymous"></script>
+    <style>
+        .wrapper {
+            min-height: calc(100vh - 15px);
+            margin: 15px auto 0;
+        }
+    </style>
+    <title>LogicPart</title>
+</head>
+
+<body cz-shortcut-listen="true">
+    <div class="container-fluid wrapper">
+        <nav class="navbar navbar-light bg-light">
+            <h1>MyToDoList</h1>

```

Comment:
can you give more descriptive name ?

Reply:
Thanks!
if I understood correctly what you mean, then I fixed)
```suggestion
 <h1>MyToDoList</h1>
 <h1>Task Tracker</h1>
```

Reply:
Thanks!
if I understood correctly what you mean, then I fixed)
```suggestion
<h1>Task Tracker</h1>
```